groclient/client.py

```diff
@@ -750,21 +750,37 @@ def get_ancestor(
         entity_id,
         distance=None,
         include_details=True,
+        ancestor_level=None,
+        include_historical=True,
     ):
-        """Given an item, metric or region, returns all its ancestors i.e.
-        entities that "contain" in the given entity
+        """Given an item, metric, or region, returns all its ancestors i.e.
+        entities that "contain" in the given entity.
+
+        The `distance` parameter controls how many levels of ancestor entities you want to be
+        returned. Additionally, if you are getting the descendants of a given region, you can
```

Comment:
typo - should be `if you are getting the ancestors`

Reply:
thanks! Updated